docs/en/solutions/How_to_use_Nodejs_on_ACP_Platform.md

```
@@ -0,0 +1,436 @@
+| darwin-arm64 (and .pkg) | macOS 15 (arm64), Xcode 16 with -mmacosx-version-min=13.5     |
+| linux-arm64             | RHEL 8 with Clang 19.1 and gcc-toolset-14-libatomic-devel[^6] |
+| linux-ppc64le           | RHEL 8 with Clang 19.1 and gcc-toolset-14-libatomic-devel[^6] |
+| linux-s390x             | RHEL 8 with Clang 19.1 and gcc-toolset-14-libatomic-devel[^6] |
+| linux-x64               | RHEL 8 with Clang 19.1 and gcc-toolset-14-libatomic-devel[^6] |
+| win-arm64               | Windows Server 2022 (x64) with Visual Studio 2022             |
+| win-x64                 | Windows Server 2022 (x64) with Visual Studio 2022             |
+
+<!--lint disable final-definition-->
+
+[^5]: Binaries produced on these systems require libstdc++12, available
+    from the [AIX toolbox][].
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "aix toolbox"

(MD052, reference-links-images)